ros2controlcli/ros2controlcli/verb/load_controller.py

```diff
@@ -21,15 +21,15 @@
 from ros2controlcli.api import add_controller_mgr_parsers, ControllerNameCompleter
 
 
-class LoadVerb(VerbExtension):
+class LoadControllerVerb(VerbExtension):
     """Load a controller in a controller manager."""
 
     def add_arguments(self, parser, cli_name):
         add_arguments(parser)
         arg = parser.add_argument('controller_name', help='Name of the controller')
         arg.completer = ControllerNameCompleter()
         arg = parser.add_argument(
-            '--state',
+            '--set_state',
```

Comment:
why to change this? and why not use the following?

```suggestion
            '--set-state',
```

Reply:
It was me misunderstanding https://github.com/ros-controls/ros2_control/issues/362 and then reverting wrongly.